gptel-org.el

```diff
@@ -417,7 +434,7 @@ non-nil (default), display a message afterwards."
    (letrec ((write-bounds
              (lambda (attempts)
                (let* ((bounds (gptel--get-buffer-bounds))
-                      (offset (caar bounds))
+                      (offset (caadar bounds))
```

Comment:
Could you add a comment here explaining the `caadar`?  I understand it but only after going and looking elsewhere (and this discussion).  It'll be useful to explain here with ane example why it needs to be `caadar`.

Reply:
f71e74b